lib/connection/pool.js

```diff
@@ -31,7 +32,7 @@ var DESTROYED = 'destroyed';
 
 var _id = 0;
 
-function supportsClusterTime(topology) {
+function hasSessionSupport(topology) {
```

Comment:
this seems to still be implementing the check for whether cluster time is supported. there is a separate description in the spec of how to check if a deployment supports sessions, which involves checking whether logicalSessionTimeoutMinutes is set. I don't see that happening anywhere, is it somehow implicitly checked?

Reply:
The check for `logicalSessionTimeoutMinutes` you reference in the spec will reside in the porcelain layer.  This checks for `$clusterTime` support, but  more importantly checks internally for whether we should look for a provided `session` on the operation - so I renamed it `hasSessionSupport`.